conf/messages

```diff
@@ -1,15 +1,30 @@
 common.service-name = Business tax account
 common.fullStop = .
+common.pageTitle = {0} - {1} - GOV.UK
+common.finish = Finish
+common.confirmAndContinue = Confirm and continue
+common.errorSummary.heading = There is a problem
+common.continue = Continue
+
+base.back = Back
 
 featureSwitch.title = Feature switches
 featureSwitch.heading = Features
 featureSwitch.submit = Submit
 
-error.browser.title.prefix = Error:
+common.error.prefixTitle = Error: {0}
 error.boolean = Please give an answer
 error.required = Please enter a value
 error.summary.title = There is a problem
 
+navTitle.nonAgent = Business tax account
+navTitle.agent = Your client’s VAT details
+navTitle.vat = VAT
+
+global.error.InternalServerError500.title = There is a problem with the service - VAT - GOV.UK
+global.error.InternalServerError500.heading = Sorry, there is a problem with the service
+global.error.InternalServerError500.message = Try again later.
```

Comment:
These 3 messages aren't being used anywhere

Reply:
Those messages overwrite some default messages within bootstrap-play. It allows us to customise the error pages that are thrown